agent/utils.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+fetch_metadata() {
+    local metadata=$(curl -s "http://169.254.169.254/metadata/$1" | jq -r .value);
+    echo $metadata
```

Comment:
Does this whole local + echo thing actually do anything? Is there any reason why you save the output of a command, just to make it the output of the next command?

Reply:
yes

Reply:
Resolving as utils is now removed